apps/workflows-dashboard/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+
+if [[ -z "$VITE_DOMAIN" ]]
+then
+    VITE_DOMAIN="localhost:3000"
+fi
+
+if [[ -z "$MODE" ]]
+then
+    MODE="development"
+fi
+
+if [[ -z "$VITE_IMAGE_LOGO_URL" ]]
+then
+    VITE_IMAGE_LOGO_URL=
+fi
+
+
+if [[ -z "$VITE_ENVIRONMENT_NAME" ]]
+then
+    VITE_ENVIRONMENT_NAME=local
+fi
+
+
+cat << EOF > /usr/share/nginx/html/config.js
+globalThis.env = {
+  VITE_API_URL: "http://$VITE_DOMAIN/api/v1/",
+  VITE_ENVIRONMENT_NAME: "$VITE_ENVIRONMENT_NAME",
+  MODE: "$MODE",
+  VITE_IMAGE_LOGO_URL: "$VITE_IMAGE_LOGO_URL",
+}
```

Comment:
Why do you need the file `config.js` if you are adding the same file content inline in here?

Reply:
i just say that you can delete config.js file